README.md

```diff
@@ -43,12 +43,12 @@ Get started with the benchmarks:
 
 <!-- PERFORMANCE_RESULTS_START -->
 
-| Server           | Requests/sec | Latency (ms) |
-| ---------------- | -----------: | -----------: |
-| [Tailcall]       |   `2,890.68` |      `34.69` |
-| [Gqlgen]         |     `935.00` |     `115.73` |
-| [Apollo GraphQL] |     `793.37` |     `128.22` |
-| [Netflix DGS]    |     `597.39` |     `191.85` |
+| Server | Requests/sec | Latency (ms) |
+|--------|--------------|--------------|
```

Comment:
Use `:` to make sure it's right aligned.

Reply:
done